discord/audit_logs.py

```diff
@@ -238,6 +238,16 @@ def _from_data(self, data):
                     'channel': self.guild.get_channel(channel_id) or Object(id=channel_id)
                 }
                 self.extra = type('_AuditLogProxy', (), elems)()
+            elif self.action.name.endswith('pin'):
```

Comment:
Move has some useful options - count and channel_id. Maybe do it here? Disconnect also has cont.

Reply:
It seems like count never goes above one, so I'd say it's safe enough to leave out unless it changes in the future.

Reply:
count definitely goes above 1.

Reply:
My testing count does go above one yes, I'll have that in the extras now.